src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/operations/live_event.py

```diff
@@ -7,28 +7,66 @@
 from azure.cli.core.commands import LongRunningOperation
 
 
-def create(client, resource_group_name, account_name, live_event_name, streaming_protocol, location,
+def create(client, resource_group_name, account_name, live_event_name, streaming_protocol, location,  # pylint: disable=too-many-locals
            auto_start=False, encoding_type=None, preset_name=None, tags=None, description=None,
-           key_frame_interval_duration=None, access_token=None, no_wait=False):
+           key_frame_interval_duration=None, access_token=None, no_wait=False, ips=None,
+           preview_locator=None, streaming_policy_name=None, alternative_media_id=None,
+           vanity_url=False, client_access_policy=None, cross_domain_policy=None, stream_options=None):
     from azure.mgmt.media.models import LiveEventInputProtocol
     from azure.mgmt.media.models import LiveEventInput
     from azure.mgmt.media.models import LiveEvent
     from azure.mgmt.media.models import LiveEventEncoding
 
-    protocol = LiveEventInputProtocol(streaming_protocol)
-    live_event_input = LiveEventInput(streaming_protocol=protocol, access_token=access_token,
+    live_event_input = LiveEventInput(streaming_protocol=LiveEventInputProtocol(streaming_protocol),
+                                      access_token=access_token,
                                       key_frame_interval_duration=key_frame_interval_duration)
 
-    live_event_encoding = LiveEventEncoding(encoding_type=encoding_type, preset_name=preset_name)
+    live_event_preview = createLiveEventPreview(preview_locator, streaming_policy_name, alternative_media_id,
+                                                ips, live_event_name)
 
-    live_event = LiveEvent(input=live_event_input, location=location, encoding=live_event_encoding)
+    policies = createCrossSiteAccessPolicies(client_access_policy, cross_domain_policy)
+
+    live_event = LiveEvent(input=live_event_input, location=location, preview=live_event_preview,
+                           encoding=LiveEventEncoding(encoding_type=encoding_type, preset_name=preset_name),
+                           tags=tags, vanity_url=vanity_url, stream_options=stream_options,
+                           cross_site_access_policies=policies)
 
     return sdk_no_wait(no_wait, client.create, resource_group_name=resource_group_name, account_name=account_name,
                        live_event_name=live_event_name, parameters=live_event, auto_start=auto_start,
-                       description=description, tags=tags, key_frame_interval_duration=key_frame_interval_duration,
+                       description=description, key_frame_interval_duration=key_frame_interval_duration,
```

Comment:
description, key_frame_interval_duration & acces_token are not expected by client.create function